### DISCUSSION TITLE
Gitlab support

### DISCUSSION THREAD

**Discussion by marksy** [lost-pixel/lost-pixel#180] [General]
Will Gitlab integration be supported?

**Comment by d-ivashchuk** [comment]
hey @marksy! GitHub first-class integration applies only to the **platform** offer. we are currently in the beta stages of all our product offerings and we decided to limit it to GitHub to have focused efforts there. As the next step of the product lifecycle of Lost Pixel, we definitely plan to support GitLab as well if we get enough confirmation that that is what users of Lost Pixel would need.

Although if you want to run Lost Pixel in the OSS mode and not use our platform it's quite easy to write a custom CI logic and use lost-pixel as a docker container. We don't have it well documented currently but if you are willing to take a shot at integrating it into your GitLab pipeline - I can assist you and then we can work on the documentation to make it clearer for anybody who wants to get the integration going there.

**Reply by DevanshAryan** [reply]
Hi, @d-ivashchuk can you guide me on how to integrate lost-pixel on gitlab, basically just wanted to know update baseline images part
